### ISSUE TITLE
[FR]: Support Excalidraw Plugin

### ISSUE THREAD

**Issue by Tiim** [ObsidianPublisher/obsidian-github-publisher#267]
### Issue validation

- [X] I checked the issue to prevent duplicate
- [X] My issue concern doesn't concern the [Mkdocs template](https://github.com/ObsidianPublisher/template-main/)

### Is your feature related to a problem ?

The [Obsidian Excalidraw](https://github.com/zsviczian/obsidian-excalidraw-plugin) plugin stores the Excalidraw drawing in a code block inside of a markdown file. If such an Excalidraw file is published, the full markdown document is published instead of the Excalidraw image.

### What solution do you want to see ?

It would be very nice if obsidian-github-publisher could detect that a file is from the Excalidraw plugin (.excalidraw.md file extension) and render the Excalidraw image to an SVG and publish this instead. When an Excalidraw file is transcluded, the SVG file could be inserted directly into the markdown.

There is a library [excalidraw-to-svg](https://github.com/JRJurman/excalidraw-to-svg) that can do this.

### Describe the alternative you've considered

An alternative is to have the Excalidraw rendering when rendering the markdown from the github repository. However this can be complex because of transcluded drawings.

### OS

Windows

### Plugin version

1.5.3

### Obsidian version & debug log

```bash session
n/a
```

**Comment by Lisandra-dev** [comment]
Why it's better than using the embedding SVG instead of the `.excalidraw` file ?

**Comment by Tiim** [comment]
Because the Excalidraw file is just a Markdown file with a code block containing json, and is not easily rendered. Especially if it is transcluded in another page.

**Comment by Lisandra-dev** [comment]
Hello, I researched and thinked but I found a problem: I didn't found any library to convert excalidraw to svg with typescript support. 

The other question is not really a problem as I just need to change the link from `.excalidraw` to `.svg` file but I change already the file path so I just need a special condition for excalidraw file.

I think I will implement this, if I found a way to use the library provided with typescript.
